app/src/main/res/xml/preferences.xml

```diff
@@ -34,6 +34,11 @@
             android:icon="@drawable/leak"
             android:title="@string/sensor_title"
             android:summary="@string/sensor_summary"/>
+        <EditTextPreference
+            android:key="minimum_accuracy"
+            android:icon="@drawable/ic_gps_fixed"
+            android:title="@string/minimum_accuracy_title"
```

Comment:
We should add a summary so we can explain to users what this setting does. Since we only accept numbers we should also set the proper `inputType` so we only get numeric characters 😃 

Should we validate that its a positive integer? 🤔

Reply:
I tried to add the summary, but it replaces the current setting, making it invisible. I don't know how to show both.

Reply:
Basically I've copied the code of session timeout. There's no any validation too ;-)

Reply:
Our conversation made me realize we had a bug around the session timeout option as the app was crashing when it got a non-numeric value #894 .  Was working on some enhancement so I placed the fix in #895

Reply:
Great. Based on your PR I've made the same changes here.